app/models/alaveteli_pro/request_summary.rb

```diff
@@ -26,6 +26,18 @@ class AlaveteliPro::RequestSummary < ActiveRecord::Base
                              "InfoRequestBatch",
                              "AlaveteliPro::DraftInfoRequestBatch"].freeze
 
+  def self.category(category_slug)
+    includes(:request_summary_categories).
+      where("request_summary_categories.slug = ?", category_slug.to_s).
+        references(:request_summary_categories)
+  end
+
+  def self.not_category(category_slug)
+    includes(:request_summary_categories).
```

Comment:
Hmm, yeah this was a bit trickier than I thought. I've fixed it in b049e05f36385b1ae7bf613102b1c5cc66386685 but I've had to resort to two queries to get it done. If you have any thoughts on how to do it more efficiently, let me know :)